dprint.json

```diff
@@ -19,7 +19,8 @@
     "**/build",
     "**/ios/Pods",
     "**/pgdata",
-    "coordinator/migrations/00000000000000_diesel_initial_setup/up.sql"
+    "coordinator/migrations/00000000000000_diesel_initial_setup/up.sql",
+    "maker/migrations/00000000000000_diesel_initial_setup/up.sql"
```

Comment:
> is there a reason we don't want these SQL files formatted?

Unfortunately formatting these queries breaks the query. 

> (diesel won't try to regenerate them after all?)

No, diesel won't regenerate this

> if we really want to skip the initial SQL migrations, then we can use something like this to make it less fragile and in a one-liner:

I do understand the convenience of this but I would prefer having them separate so that it's clear which files are ignored.